docs/debugging.md

```diff
@@ -0,0 +1,67 @@
+# CLI Debugging Guide
+
+This guide will help you debug issues with your Dependabot update using the Dependabot CLI.
+
+## Getting started
+
+First, test to make sure you have a working Dependabot CLI by performing a simple update, like `dependabot update --dry-run go_modules rsc/quote -o out.yml`. This should complete without error, and you can examine the out.yml file which should contain two calls to `create_pull_request`.
```

Comment:
> dependabot update --dry-run go_modules rsc/quote

Any reason for this specific repo? I wonder if we could fork it or create something similar in the `dependabot` org so that we can be sure that it doesn't change 🤔

Reply:
It's a repo created to demo Go versioning, so I feel like it's ok. 

Also slightly worried if the word "dependabot" appears in the example again it might be confusing: `dependabot update --dry-run go_modules dependabot/quote`